unyt/unit_registry.py

```diff
@@ -39,6 +40,15 @@ def _sanitize_unit_system(unit_system, obj):
     return unit_system_registry[str(unit_system)]
 
 
+@lru_cache
+def cached_sympify(u):
+    """
+    Successive loads of unit systems produce the same calls to sympify in UnitRegistry.from_json
+    Even within a single load, this is a net gain because there will often be a few cache hits
```

Comment:
`flake8` is complaining that these lines are too long. In general python docstrings should be less than 72 characters wide so they fit comfortable in an 80 character wide terminal.